prow/proxy-common.inc

```diff
@@ -43,15 +43,16 @@ fi
 export BAZEL_OUT="$(bazel info ${BAZEL_BUILD_ARGS} output_path)/k8-fastbuild/bin"
 
 # Use GCP service account when available.
-if [[ -n "${GOOGLE_APPLICATION_CREDENTIALS:-}" ]]; then
-  echo "Detected GOOGLE_APPLICATION_CREDENTIALS, activating..." >&2
-  gcloud auth activate-service-account --key-file="${GOOGLE_APPLICATION_CREDENTIALS}"
-
-  # Use RBE when logged in.
-  BAZEL_BUILD_RBE_INSTANCE="${BAZEL_BUILD_RBE_INSTANCE:-projects/istio-testing/instances/default_instance}"
-  BAZEL_BUILD_RBE_JOBS="${BAZEL_BUILD_RBE_JOBS:-200}"
-  if [[ -n "${BAZEL_BUILD_RBE_INSTANCE}" && "${BAZEL_BUILD_RBE_JOBS}" -gt 0 ]]; then
-    echo "Using RBE: ${BAZEL_BUILD_RBE_INSTANCE}"
-    export BAZEL_BUILD_ARGS="${BAZEL_BUILD_ARGS} --google_credentials=${GOOGLE_APPLICATION_CREDENTIALS} --config=remote-clang-libc++ --config=remote-ci --remote_instance_name=${BAZEL_BUILD_RBE_INSTANCE} --jobs=${BAZEL_BUILD_RBE_JOBS}"
-  fi
-fi
+# Disabling RBE due to failures like https://prow.istio.io/view/gcs/istio-prow/pr-logs/pull/istio_proxy/2633/release-test_proxy/211
```

Comment:
`export BAZEL_BUILD_RBE_JOBS=0` works as well, without the need to comment out everything.

Reply:
Also, you could disable it only for the release build, and not all.